src/main/webapp/app/tasks/task-list/task-list.component.ts

```diff
@@ -12,17 +16,63 @@ import { TaskService } from '../task.service';
   styleUrls: ['./task-list.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class TaskListComponent {
-
+export class TaskListComponent implements OnInit {
   @Input() tasks: Task[] | null = null;
-
   @Output() deleted: EventEmitter<Task> = new EventEmitter();
+  @Output() updated: EventEmitter<Task> = new EventEmitter();
+  @ViewChild('picker') picker: any;
+  public disabled = true;
+  public dateSelected = ""
+  public timeSelected = ""
+  public disabledDateTime = false;
+  public showSpinners = true;
+  public showSeconds = false;
+  public touchUi = false;
+  public enableMeridian = false;
+  public minDate: Date = new Date();
+  public stepHour = 1;
+  public stepMinute = 1;
+  public stepSecond = 1;
+  public color: ThemePalette = 'primary';
+  
+  constructor(@Inject('TaskService') private taskService: TaskService,
+   public dialog: MatDialog) {
+  }
 
-  constructor(@Inject('TaskService') private taskService: TaskService) {}
+  ngOnInit() {
+    this.minDate = Common.setMinDate()
+  }
 
   delete(task: Task): void {
     this.taskService.delete(task.id).subscribe(() => {
       this.deleted.emit(task);
     });
   }
+
+ 
+  getObject(task: Task) {
```

Comment:
I guess a more speaking name would be benefitial here

Reply:
yes it should be getDueDate()